src/reducers/index.js

```diff
@@ -2,5 +2,6 @@ import { combineReducers } from 'redux';
 import { charsReducer } from './starWarsReducer';
 
 export default combineReducers({
-  charsReducer
+  // I spent a very long time tryng to debug this project. It turns out it was set up wrong in the first place. In addition to a change in CharacterList.js, it needed 'characters: ' here:
+  characters: charsReducer
```

Comment:
hm, this is how you'd have to write it based on how you referenced the data in the CharacterListView mapStateToProps function, but it does cause some semantic silliness since you access the list of characters in CharacterList with `props.characters.characters`. My suggestion: leave this as just `charsReducer` here. Then when accessing the store data in your mapStateToProps, you'd reference it with `state.charsReducer.characters` instead, and then in your CharacterList just as `props.characters`. This puts the extra object notation inside your MSTP instead, but if you have multiple reducers being passed in to the combineReducer function, it forces you to explicitly reference the actual reducer name `charsReducer` (or whichever other reducer you might need to reference) and avoids other issues, like the need to reference the characters with props.characters.characters